src/Microsoft.DotNet.ProjectModel/OutputPathCalculator.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Should we change the name of this class into a more generic PathCalculator? Or Output is still very descriptive?